bundles/WebToPrintBundle/src/Installer.php

```diff
@@ -0,0 +1,126 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\WebToPrintBundle;
+
+use Doctrine\DBAL\ArrayParameterType;
+use Doctrine\DBAL\Connection;
+use Pimcore\Db;
+use Pimcore\Extension\Bundle\Installer\SettingsStoreAwareInstaller;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Installer extends SettingsStoreAwareInstaller
+{
+
+    protected const USER_PERMISSION_CATEGORY = 'Pimcore Web2Print Bundle';
+    protected const USER_PERMISSIONS = [
+        'web2print_settings',
+    ];
+
+    protected const STANDARD_DOCUMENT_ENUM_TYPES = [
+        'page',
+        'link',
+        'snippet',
+        'folder',
+        'hardlink',
+        'email',
+        'newsletter'
+    ];
+
+    protected const BUNDLE_EXTRA_DOCUMENT_ENUM_TYPES = [
+        'printpage',
+        'printcontainer'
+    ];
+
+    public function install(): void
+    {
+        $this->installDatabaseTable();
+        $enums = array_unique(array_merge($this->getCurrentEnumTypes(), self::BUNDLE_EXTRA_DOCUMENT_ENUM_TYPES));
+        $this->modifyEnumTypes($enums);
+        $this->addUserPermission();
+        parent::install();
+    }
+
+    public function uninstall(): void
+    {
+        // Only remove permissions. Cleanup can be done by dev or command
+        $this->output->writeln("<info>Uninstalling only removes permissions. To clean up all documents and dependencies, please run 'bin/console pimcore:documents:cleanup printpage printcontainer'</info>");
```

Comment:
This doesn't show up but could be replaced with an `echo` for the moment. Maybe add some `#####` to highlight it a little bit

Reply:
I actually just created a Consoleoutput like this.
` $output = new ConsoleOutput(OutputInterface::VERBOSITY_NORMAL, true);`